tests/step6_file.mal

```diff
@@ -86,9 +86,20 @@
 ;=>9
 
 ;>>> deferrable=True
-;>>> optional=True
 ;;
-;; -------- Deferrable/Optional Functionality --------
+;; -------- Deferrable Functionality --------
+
+;; Testing reading of large files
+(load-file "../tests/computations.mal")
+;=>nil
+(sumdown 2)
+;=>3
+(fib 2)
+;=>1
+
+;>>> deferrable=True
```

Comment:
I think that optional=True accidentally became deferrable=True on this line.

Reply:
That's true. Corrected.